app/controllers/assignments_controller.rb

```diff
@@ -252,6 +255,13 @@ def create
 
   def summary
     @assignment = record
+    @lti_deployments = LtiLineItem.where(assessment_id: @assignment.id)
+                                  .joins(:lti_deployment)
+                                  .joins(lti_deployment: :lti_client)
+                                  .pluck_to_hash('lti_deployments.id',
+                                                 'lti_clients.host',
+                                                 'lti_deployments.lms_course_name')
+    @lti_deployments.each { |deployment| deployment.deep_transform_keys! { |key| key.to_s.split('.')[-1] } }
```

Comment:
I'm pretty sure you can just use `transform_keys!` here because none of the hashes are nested